### ISSUE TITLE
Should this work with Minecraft v1.7.1 on Windows 10?

### ISSUE THREAD

**Issue by stussey** [The-HeX/mcpe-geometry-generator#5]
Should this work with Minecraft v1.7.1 on Windows 10?
When try _**/wsserver 127.0.0.1:12112**_ I get a message stating that it cannot connect. Am I missing something?

**Comment by stussey** [comment]
I figured out my issue. Had issues with my network address.

**Comment by erichexter** [comment]
Any information you can share, may help me add more information to the
FAQ.  Sometime, I have to try the command more then once for it to connect.

**Comment by stussey** [comment]
Thanks Eric,
I had an issue with my network stack. Once I got that resolved everything started working correctly.

**Comment by erichexter** [comment]
It looks like you can read the code, there are a few other command
available that are not documented. If you have a log file of create
commands you can replay them using the script command. I did this to test
out a build, then replay it in a different world.
https://github.com/The-HeX/minecraft-worldedit-bedrock/blob/master/src/WorldEdit/Program.cs#L62

**Comment by stussey** [comment]
That's cool. I'm trying to get access to the 1.8 beta for Minecraft on
Windows 10. I played with your code  for a while about 7 months ago. I lost
interest in Minecraft for a while and just started to find new interest. I
had modified the saved position handler to use sqlite to store the saved
positions as well as creating a new handler to allow admins to create item
packs and give them to players. I have a renewed interest and decided to
pick everything up and finish the projects I've started.


[image: Mailtrack]
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality5&>
Sender
notified by
Mailtrack
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality5&>
11/18/18,
4:50:01 AM
